libs/fluid-elements/design-system-provider/jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
please do testing with bazel ther you need only a `jest.config.json` with a name inside

Reply:
Ah I thought Bazel testing didn't work with web components yet 👍